packages/strapi-provider-email-nodemailer/LICENSE.md

```diff
@@ -0,0 +1,8 @@
+Copyright (c) 2018 Automat-IT
```

Comment:
This needs to be changed over to our license, see: https://github.com/strapi/strapi/blob/master/packages/strapi-provider-email-amazon-ses/LICENSE as an example